tools/telemetry-generator/src/handlers/stageTimingRetriever.ts

```diff
@@ -54,8 +54,17 @@ module.exports = function handler(fileData, logger) {
 		if (job.stageName === "runAfterAll") {
 			continue;
 		}
+
+		// Transfer the event StageTiming (associated with the stress, e2e and tests performance) to the
+		// table suffixed with performance_tests. For telemetry related to the build or other events, retain
+		// them in the original table.
+		const category =
```

Comment:
shouldn't this always be performance_tests. also, i don't really like this method of segmenting the data, as it break common querying scenarios, like `union *_ffautomation_*`. for this reason change the namespace would be better

Reply:
I would also prefer a different namespace. And if we don't do that, then I'd suggest we find a better name for the category, it's not really (only) about "tests". I had suggested "engineering", since it would encompass different kinds of data from our engineering systems (pipeline status, performance test measurements, bundle size measurements) but I don't particularly like it, just couldn't come up with a better idea. "ffengineering" as namespace doesn't feel as awkward to me for some reason.

Reply:
I believe this method can differentiate between tasks with stage names containing `e2e`/`test` and those with `build`/`publish_npm_*`. The latter will be stored in the old performance table, so it won't always fall under performance_tests. But yes using a namespace seems indeed a better approach. I will update that.